src/applications/caregivers/config/chapters/primary/contactInformation.js

```diff
@@ -1,9 +1,8 @@
-// import fullSchema from 'vets-json-schema/dist/10-10CG-schema.json';
+import fullSchema from 'vets-json-schema/dist/10-10CG-schema.json';
 import { titleUI } from 'platform/forms-system/src/js/web-component-patterns';
 import { emailUI, phoneUI } from '../../../definitions/sharedUI';
 import CaregiverContactInfoDescription from '../../../components/FormDescriptions/CaregiverContactInfoDescription';
 import content from '../../../locales/en/content.json';
-import fullSchema from '../../10-10CG-schema.json';
```

Comment:
After looking at all the places this needed updated, should we consider some kind of shared config file that we could import instead of directly importing it in each one? Maybe group the content.json import into that as well? I'm not as familiar with the organization of the front end but just a thought.

Reply:
Updated this to utilize the imports file as you suggested!